surfactantplugin-syft/surfactantplugin_syft.py

```diff
@@ -0,0 +1,98 @@
+import json
+import subprocess
+import time
+from typing import List, Optional
+
+import surfactant.plugin
+from surfactant.plugin.manager import get_plugin_manager
+from surfactant.sbomtypes import SBOM, Relationship, Software
+
+
+@surfactant.plugin.hookimpl
+def extract_child_info(
```

Comment:
```suggestion
def extract_file_info(
```
Switch to using the existing hook.